### PR TITLE
update playground tutorials

### LINKED ISSUE #296: Enums
## Context

Add support for enums, which:
- Don't have associated value(s) yet
- Don't have a raw value yet

In the future this feature can be extended, for example, add associated values and a raw value.

Example:
```swift
enum Color {
  case red
  case green
  case blue
}
```

## Definition of Done

- [x] Add AST element
- [x] Parse
- [x] Check
- [x] Interpret
- [x] Tests
- [x] Documentation

### PR DESCRIPTION
**Pull request by MrDSGC**
Closes onflow/flow-playground#296

## Description

- update playground links to new tutorials with syntax fixes
- fixed formatting for playground links to be more consistant

______

### CODE REVIEW

**Comment by MrDSGC** [comment]
List of updated Playground links

03
old    https://play.onflow.org/a26ec672-14c8-42cd-85ad-1409a8a23a84
new   https://play.onflow.org/b70199ae-6488-4e58-ae58-9f4ffecbd66a

04
old   https://play.onflow.org/3a541312-a231-4f11-be79-23bcd2d94a6c
new  https://play.onflow.org/a7f45bcd-8fda-45f6-b443-4b77302a1687

05
old  https://play.onflow.org/ae2f2a83-6698-4e03-93cf-70d35627e28e
new https://play.onflow.org/a21087ad-b22c-4981-b49e-17297e916fa6

07 & 08
old  https://play.onflow.org/45ae690e-c527-409c-970e-57f03df92790
new  https://play.onflow.org/49ec2856-1258-4675-bac3-850b4bae1929

09
old  https://play.onflow.org/75a6ce58-74bb-468f-b7bd-425716c93c3e
new  https://play.onflow.org/d120f0a7-d411-4243-bc59-5125a84f99b3
  
10
old  https://play.onflow.org/cda4d057-5898-4a7f-be34-fd47ec75085f
new  https://play.onflow.org/01f812d7-799a-42fd-b9cb-9ffe556e02ad

**Comment by MrDSGC** [comment]
@joshuahannan should be ready for review

File: docs/tutorial/05-non-fungible-tokens-2.mdx [hunk 1]
```diff
@@ -29,22 +29,21 @@ a full implementation for **Non-Fungible Tokens (NFTs)**.
...
-Open the starter code for this tutorial in the Flow Playground:
-
-<a href="https://play.onflow.org/af553a60-7b73-4e2e-b145-80a7c7e088dc" target="_blank">
-  https://play.onflow.org/af553a60-7b73-4e2e-b145-80a7c7e088dc
-</a> <br/>
-The tutorial will ask you to take various actions to interact with this code.
-
+  Open the starter code for this tutorial in the Flow Playground:
+  <a 
+    href="https://play.onflow.org/af553a60-7b73-4e2e-b145-80a7c7e088dc"
```
Comment (joshuahannan) [inline docs/tutorial/05-non-fungible-tokens-2.mdx:34]: It looks like this link is the same as the old one. Did you not have to make any changes to it?

File: docs/tutorial/10-resources-compose.mdx [hunk 2]
```diff
@@ -7,19 +7,21 @@ In this tutorial, we're going to walk through how resources can own other resour
...
-Open the starter code for this tutorial in the Flow Playground:
-<a href="https://play.onflow.org/cda4d057-5898-4a7f-be34-fd47ec75085f" target="_blank">https://play.onflow.org/cda4d057-5898-4a7f-be34-fd47ec75085f</a><br/>
-The tutorial will be asking you do take various actions to interact with this code.
-
+  Open the starter code for this tutorial in the Flow Playground:
+  <a 
+    href="https://play.onflow.org/01f812d7-799a-42fd-b9cb-9ffe556e02ad"
+    target="_blank"
+  >
+    https://play.onflow.org/01f812d7-799a-42fd-b9cb-9ffe556e02ad
```
Comment (joshuahannan) [inline docs/tutorial/10-resources-compose.mdx:15]: I checked this link and it looks like the code still doesn't have the updated code that works with secure cadence in the 2.Tip Hat transaction. Can you make sure that that is updated and you have the correct link here?

**Review by joshuahannan — COMMENTED** [review]
A few comments. Also, I clicked on a few of the tutorial links and saw that the links that they have in their first contracts that go to the tutorial pages are broken because they are still using the old docs site link. Can you make sure those are all updated also? Thank you!

File: docs/tutorial/05-non-fungible-tokens-2.mdx (hunk 1 above)
Comment (MrDSGC) [reply docs/tutorial/05-non-fungible-tokens-2.mdx:34]: No changes. I was just updating the formatting to match the other tutorials

File: docs/tutorial/10-resources-compose.mdx [hunk 3]
```diff
@@ -99,7 +103,7 @@ pub contract KittyVerse {
                 return "Howdy Y'all"
             } else if self.name == "Top Hat" {
                 return "Greetings, fellow aristocats!"
-            }
+            }
```
Comment (robert-e-davidson3) [inline docs/tutorial/10-resources-compose.mdx:106]: random trailing space?
```suggestion
            }
```

File: docs/tutorial/10-resources-compose.mdx [hunk 4]
```diff
@@ -70,16 +71,19 @@ Open the account `0x01` tab which has the contract named `KittyVerse.cdc`. Deplo
...
+// KittyVerse.cdc
+//
 // The KittyVerse contract defines two types of NFTs.
 // One is a KittyHat, which represents a special hat, and
 // the second is the Kitty resource, which can own Kitty Hats.
 //
 // You can put the hats on the cats and then call a hat function
 // that tips the hat and prints a fun message.
-//
+//
```
Comment (robert-e-davidson3) [inline docs/tutorial/10-resources-compose.mdx:82]:
```suggestion
//
```

**Review by robert-e-davidson3 — APPROVED** [review]

**Review by SupunS — APPROVED** [review]

**Pull request merged**